halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/converter/LocalFileConverter.java

```diff
@@ -18,12 +18,26 @@
 package com.netflix.spinnaker.halyard.cli.command.v1.converter;
 
 import com.beust.jcommander.IStringConverter;
+import com.netflix.spinnaker.halyard.core.GlobalApplicationOptions;
+import com.netflix.spinnaker.halyard.core.error.v1.HalException;
+import com.netflix.spinnaker.halyard.core.problem.v1.Problem;
+import org.aspectj.util.FileUtil;
 
 import java.io.File;
+import java.io.IOException;
+
+public class LocalFileConverter implements IStringConverter<String> {
 
-public class PathExpandingConverter implements IStringConverter<String> {
   @Override
   public String convert(String value) {
+    if (GlobalApplicationOptions.getInstance().isUseRemoteDaemon()) {
+      try {
+        return FileUtil.readAsString(new File(value));
+      } catch (IOException e) {
+        throw new HalException(Problem.Severity.FATAL,
+            "Config references file that is unreadable: " + value);
```

Comment:
I don't think this should read "config", instead "Was passed parameter " + value + " to unreadable file: " + e.getMessage()

Reply:
Done.